ios/FluentUI/Navigation/SearchBar/SearchBarTokenSet.swift

```diff
@@ -176,7 +180,8 @@ extension SearchBarTokenSet {
 public extension SearchBar {
     @objc(MSFSearchBarStyle)
     enum Style: Int {
-        case lightContent
-        case darkContent
+        case canvas
```

Comment:
should it be onCanvas, onBrandBackground, etc? we should add documentation too for each style

Reply:
onHosted sounds wrong - how do you feel about onNavigationBar?

Reply:
onNavigationbar still might not be clear because navigationbar could be "white/grey" or "branding/grey"

Reply:
Do you have a suggestion for a different name? onHosted and onNavigationBar both are out, sounds like.

Reply:
I have already tested with navigation controller demo. I will add screenshots.

I will update the styles.